build_tools/docker/manage_images.py

```diff
@@ -252,27 +259,27 @@ def parse_prod_digests() -> Dict[str, str]:
                     "--file",
                     image_path,
                     "--tag",
-                    tagged_image_url,
+                    image_url,
                     ".",
                 ],
-                dry_run=args.dry_run,
+                dry_run=dry_run,
             )
 
-            utils.run_command(
-                ["docker", "push", tagged_image_url], dry_run=args.dry_run
-            )
+            utils.run_command(["docker", "push", image_url], dry_run=dry_run)
 
-            digest = get_repo_digest(tagged_image_url, args.dry_run)
+            image_digest = get_repo_digest(image_url, dry_run)
 
-            # Check that the image is in "prod_digests.txt" and append it to the list
-            # in the file if it isn't.
+            # Check that the image is in "prod_digests.txt" and append it to the
```

Comment:
Probably should update the comment to not hardcode the filename also

Reply:
Done.